File: Decorator/Java/CoffeeShopNoDecorator/src/CoffeeShopNoDecorator/DarkRoastWithSoy.java
package CoffeeShopNoDecorator;

public class DarkRoastWithSoy extends Beverage
{
    public DarkRoastWithSoy()
    {
        description = "Dark roast with soy milk";            
    }

    @Override
    public double getPrice()
    {
        return 1.1d;
    }
}

File: Decorator/C#/CoffeeShopNoDecorator/DarkRoastWithSoy.cs
namespace CoffeeShopNoDecorator
{
    public class DarkRoastWithSoy :  Beverage
    {
        public DarkRoastWithSoy()
        {
            Description = "Dark roast with soy milk";            
        }

        public override decimal GetPrice()
        {
            return 1.1m;
        }
    }
}